core/dbsecurity/auth/src/authEvents.cpp

```diff
@@ -0,0 +1,134 @@
+// @@@ START COPYRIGHT @@@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// @@@ END COPYRIGHT @@@
+
+#include "authEvents.h"
+#include <sys/time.h>
+#include <str.h>
+#include <unistd.h> 
+#include <stdio.h>
+#include <iostream>
+#include "seabed/ms.h"
+#include "seabed/fserr.h"
+
+static std::string AUTH_COMPONENT = "DBSECURITY";
+std::vector<AuthEvent> authEvents;
```

Comment:
Changed it to not be a global, passing the structure to all component that may generate events.